packages/components/src/Select/Select.types.ts

```diff
@@ -0,0 +1,43 @@
+import { CommonFormFieldProps, FormFieldProps } from "../FormField";
+
+export interface SelectProps
+  extends Pick<
+      CommonFormFieldProps,
+      | "id"
+      | "align"
+      | "description"
+      | "disabled"
+      | "invalid"
+      | "inline"
+      | "name"
+      | "onValidation"
+      | "placeholder"
+      | "size"
+    >,
+    Pick<
+      FormFieldProps,
+      | "autofocus"
+      | "onEnter"
+      | "onBlur"
+      | "onFocus"
+      | "inputRef"
+      | "wrapperRef"
+      | "validations"
+      | "children"
+      | "prefix"
+      | "suffix"
+    > {
+  readonly defaultValue?: string | number;
```

Comment:
I noticed that this breaks V1 usages because the initial implementation to `Select` also used `FormFieldProps` which could be a `Date` as well

Reply:
This also applies to value and `onChange` (I would override these in the V2 version) and keep the original ones that were from `FormFieldProps`